### PR TITLE
Rename EnableMobileData -> TurnOnMobileData

### PR DESCRIPTION
**Pull request by jorunfa**
To make the difference between what the feature does and whether or not
it is enabled more clear.

### CODE REVIEW

File: connect/src/com/telenor/connect/ConnectSdk.java [hunk 1]
```diff
@@ -411,7 +412,7 @@ public static String getClientId() {
 
     public static boolean showMobileDataDialog() {
```
Comment (SerhiiBovtriuk) [inline connect/src/com/telenor/connect/ConnectSdk.java]: I guess method also needs to be renamed.

File: connect/src/com/telenor/connect/utils/TurnOnMobileDataAnalytics.java [hunk 2]
```diff
@@ -7,14 +7,14 @@
 
 import java.io.Serializable;
 
-public class EnableMobileDataDialogAnalytics implements Serializable {
+public class TurnOnMobileDataAnalytics implements Serializable {
```
Comment (SerhiiBovtriuk) [inline connect/src/com/telenor/connect/utils/TurnOnMobileDataAnalytics.java]: Maybe it should be "TurnOnMobileData**Dialog**Analytics"?

File: connect/src/com/telenor/connect/ConnectSdk.java (hunk 1 above)
Comment (jorunfa) [reply connect/src/com/telenor/connect/ConnectSdk.java]: Fixed in e3fa8e2.

Comment (jorunfa) [reply connect/src/com/telenor/connect/ConnectSdk.java]: Good catch.

File: connect/src/com/telenor/connect/utils/TurnOnMobileDataAnalytics.java (hunk 2 above)
Comment (jorunfa) [reply connect/src/com/telenor/connect/utils/TurnOnMobileDataAnalytics.java]: It was intentional, but I think perhaps I've changed my mind and agree with you. The dialog is core to the feature.

**Pull request merged**